norbit-webapp/pages/index.tsx

```diff
@@ -1,11 +1,44 @@
 import Head from 'next/head';
 import styles from '../styles/Home.module.css';
 import Dashboard from 'components/dashboard/Dashboard';
+import { Amplify, Auth } from 'aws-amplify';
+import { amplifyConfig } from 'amplify-config';
+import { AWSIoTProvider } from '@aws-amplify/pubsub/lib/Providers';
+import { useState } from 'react';
+import { TLightIntensityData } from 'components/LightIntensityComponent/types';
+import { useSubscribeToTopics } from 'utils/useSubscribeToTopic';
+import {useEffect}  from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'redux/store';
+import { userSignedIn } from '@redux/slices/amplifySlice';
+import { setUserName } from '@redux/slices/amplifySlice';
```

Comment:
These could be inlined:
```suggestion
import { userSignedIn, setUserName } from '@redux/slices/amplifySlices';

```